tests/ui/block-network-request.goml

```diff
@@ -0,0 +1,13 @@
+// This test is meant to ensure that we can block an external script request.
+// Since we are intentionally causing request errors, don't block them.
+fail-on-request-error: false
```

Comment:
Could you mention this in the documentation as well please? Could be surprising that a test fails if you block a request in another command.